Sources/StreamChatUI/UIConfig+MessageList.swift

```diff
@@ -12,6 +12,10 @@ public extension _UIConfig {
         public var defaultMessageCell: _СhatMessageCollectionViewCell<ExtraData>.Type =
             _СhatMessageCollectionViewCell<ExtraData>.self
 
+        /// A `UIView` subclass which serves as container for `typingIndicator` and `UILabel` describing who is currently typing
+        public var typingIndicatorOverlayView: _ChatMessageListTypingIndicatorView<ExtraData>.Type =
+            _ChatMessageListTypingIndicatorView<ExtraData>.self
+
```

Comment:
This should be in the root https://www.notion.so/UI-Config-Rules-Re-Think-5f58357b1f324879927aaeea1aa980a0 + the name of the variable is off